s2/bq/test/s2_integration.js

```diff
@@ -0,0 +1,141 @@
+const assert = require('assert').strict;
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const BQ_PROJECTID = process.env.BQ_PROJECTID;
+const BQ_DATASET_S2 = process.env.BQ_DATASET_S2;
+
+describe('S2 integration tests', () => {
+
+    let client;
+    before(async () => {
+        if (!BQ_PROJECTID) {
+            throw "Missing BQ_PROJECTID env variable";
+        }
+        if (!BQ_DATASET_S2) {
+            throw "Missing BQ_DATASET_S2 env variable";
+        }
+        client = new BigQuery({projectId: `${BQ_PROJECTID}`});
+    });
+
+    it ('Returns the proper version', async () => {
+        const query = `SELECT \`${BQ_PROJECTID}\`.\`${BQ_DATASET_S2}\`.VERSION() as versioncol;`;
+        let rows;
+        await assert.doesNotReject( async () => {
+            const [job] = await client.createQueryJob({ query: query });
+            [rows] = await job.getQueryResults();
+        });
+        assert.equal(rows.length, 1);
+        assert.equal(rows[0].versioncol, 1);
+    });
+
+    it ('KEY / ID conversions should work', async () => {
```

Comment:
Testing compatiblity is great, but this should also check static values to confirm they are correct.  Also it should include a test for https://github.com/CartoDB/carto-spatial-extension/issues/29